lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,38 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: ['derpiboo.ru'],
+	matchRe: /^https?:\/\/(?:www\.)?derpiboo.ru\/\d+$/i,
```

Comment:
Should probably allow query strings or hashes after the id, maybe something like `/^https?:\/\/(?:www\.)?derpiboo.ru\/\d+(?:$|[?#])/i`.